paddlecloud/notebook/tls.py

```diff
@@ -36,6 +36,7 @@ def create_user_cert(ca_path, username):
         %s/%s.csr -subj \"/CN=%s\""%\
         (user_cert_dir, username,
         user_cert_dir, username, username))
+    user_cert_cmds.append("rm -f %s/ca.srl" % settings.USER_CERTS_PATH)
```

Comment:
In my own curiosity, why delete this file, or could you add some comments here?

Reply:
Done.